src/game/header/hamburger/hamburger.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import s from "./hamburger.module.css";
+
+export function Hamburger(props) {
+	let hamburgerStatus = props.isMenuOpen;
+	let hamburgerClass = hamburgerStatus ? (`${s.hamburger} ${s.active}`) : (`${s.hamburger}`);
```

Comment:
you can use ternary operator inside comparison, simplify the code

Reply:
Should it look like this?

```suggestion
let hamburgerClass = `${styles.hamburger} ${props.isMenuOpen ? styles.active : ''}`;
```

Reply:
Yep!